backstop/scenarios/modal.js

```diff
@@ -7,4 +7,10 @@ module.exports = [
     actions: [click('#open-modal'), hover('body')],
     selectors: ['document'],
   }),
+  scenario('full-page-modal', {
+    // Hovering `body` to avoid random test failures
+    // when "Open modal" sometimes is and sometimes is not hovered.
+    actions: [click('#open-full-page-modal'), hover('body')],
+    selectors: ['document'],
```

Comment:
```suggestion
    selectors: ['document'],
    hideSelectors: ['h1', 'hr', 'theme-button', '#open-modal', '#open-full-page-modal'],
```
Can you add this `hideSelectors` part to both scenarios?  It will hide the non-relevant stuff from the test images.

Reply:
Done